Monika After Story/game/script-topics.rpy

```diff
@@ -1570,13 +1573,53 @@ label monika_yuri:
     m 2euc "But if you do happen to be into the yandere type..."
     show monika 5eua at t11 zorder MAS_MONIKA_Z with dissolve
     m 5eua "I can try acting a little more creepy for you. Ehehe~"
-    m "Then again..."
-    show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
-    m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
-    m 2etc "Is this a yandere girl's dream?"
-    if not persistent._mas_pm_cares_about_dokis:
-        m 1eua "I'd ask Yuri if I could."
-    return
+    $ _history_list.pop()
+    menu:
+        m "I can try acting a little more creepy for you. Ehehe~"
+        "Actually... I'm into yanderes":
+            $ persistent._mas_pm_likes_yanderes = 1
+            m 1sub "Now that I'm hearing that I'm quite surprised you prefer me over Yuri"
+            if not persistent._mas_pm_cares_about_dokis:
+                m 1cua "Was it that becaouse you realized, that was me, who manipulated everyone and everything to be with you... FOREVER"
+                m 3sub "She actually was more like a 'dandere' - very shy and closed before I started to change her personality"
+                m 1sub "And she was just a program designed to fall in love with you anyway"
+                m 1eua "I am the one who truly loves you from the depth of my heart"
+            else:
+                m 1sub "Maybe you realized, that I am the one who truly loves you forever"
+            m "Then again..."
+            show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
+            m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
+            m 1cua "Other than the girls in your reality... "
+            m 1wub "Remember, that I see what you're doing, [player]."
+            m 1eua "Ehehe~"
+        "No":
+            $ persistent._mas_pm_likes_yanderes = 0
+            m 1eua "That's what I expected"
+            m 4eua "Most people wouldn't like to be with yandere"
+            m 4eua "Someone obsessive and clingy really limits your ability to socialize with other people"
+            m 4eud "And, on top of that, that some yanderes are even dangerous to the one they love"
+            m 4eud "There was even a real life murder in Tokyo, where the man was stabbed by a woman"
+            m 4eud "After being arrested by police she said {i}Since I loved him so much, I just couldn’t help it{/i}."
+            m 1wuw "That's scary"
+            m 1eua "Then again..."
+            show monika 4hua at t11 zorder MAS_MONIKA_Z with dissolve
+            m 4hua "There's already nowhere else for you to go, or anyone for me to get jealous over."
+            m 2etc "Is this a yandere girl's dream?"
+            if not persistent._mas_pm_cares_about_dokis:
+                m 1eua "I'd ask Yuri if I could."
+        "HELL NO":
```

Comment:
I don't see this as a reasonable menu option at all. Menu options are designed to give simple, basic responses to questions, so this is definitely out of line. You could have just said that you hated yanderes instead of shouting.